addons/beehave/nodes/composites/sequence_random.gd

```diff
@@ -3,28 +3,23 @@
 ## will be executed in a random order.
 @tool
 @icon("../../icons/sequence_random.svg")
-class_name SequenceRandomComposite extends Composite
+class_name SequenceRandomComposite extends RandomizedComposite
 
 
+signal reseted(new_order)
```

Comment:
strange name btw, something like `state_reset` or just `reset` might be better.

Reply:
Oh dont mind that I just did that for testing purposes... (like that node_2d.tscn)
But now that you mentioned I think this will be needed for the unit tests.

Reply:
Another question: there are a couple of tests that I would need a scene (ie.: updating the weights when nodes are added in runtime) but theres only one scene test `e2e_test`. In terms of folder structure, where could I keep the new tests?

Reply:
This looks good how you did it! Basically with the node. `e2e_test` was just something to check that the addon generally loads.